### PR TITLE
fixing bug where map breaks on first load

### LINKED ISSUE #160: Map javascript on Safari sometimes not loading fast enough-- results in static SVG map with no interactability.
screen shot 2015-04-17 at 3 54 03 pm

### PR DESCRIPTION
**Pull request by evantarrh**
The problem was that even though we were checking to make sure the SVG could be resolved to a variable, not all of the elements of the SVG had actually loaded yet. This adds another check before running the JS to make sure that the SVG is 100% loaded.

closes #160 for real this time.

### CODE REVIEW

**Comment by jzf2101** [comment]
:+1:  yay

**Comment by RaymondXu** [comment]
:+1:

File: density/static/js/map.js [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 document.addEventListener('DOMContentLoaded', function() {
-  console.log(document.getElementById('map'));
-
   // This recursive function waits until the SVG is fully loaded
   // before executing the relevant javascript. This solves
   // the issue of the javascript not being run on Safari.
+
   function checkReady() {
     if (document.getElementById('map').getSVGDocument() == null) {
       setTimeout(checkReady, 5);
```
Comment (RaymondXu) [inline density/static/js/map.js]: We should make 5 a variable.

File: density/static/js/map.js [hunk 2]
```diff
@@ -75,6 +76,7 @@ document.addEventListener('DOMContentLoaded', function() {
 
       var percent = (totalFullness / numFloorsInBuilding); //Should be from [0, 100]
       var buildingElement = innerSvg.getElementById('parent_' + building); //e.g. parent_43
+
       if(buildingElement != null) {
         buildingElement.style.opacity = percent / 100;
       }
```
Comment (RaymondXu) [inline density/static/js/map.js]: I can't comment on the old stuff, but i think there should be a space surrounded the division sign in  `(popup.offsetWidth/2)).toString()`

**Comment by evantarrh** [comment]
thanks @RaymondXu! good calls

**Pull request merged**